src/main/java/yalter/mousetweaks/forge/ConfigGui.java

```diff
@@ -120,12 +129,14 @@ public void onGuiClosed() {
 		Main.config.lmbTweakWithoutItem = lmbTweakWithoutItem.getBoolean();
 		Main.config.wheelTweak = wheelTweak.getBoolean();
 		Main.config.wheelSearchOrder = wheelSearchOrder.getString().equals("First to last")
-		                               ? WheelSearchOrder.FIRST_TO_LAST
-		                               : WheelSearchOrder.LAST_TO_FIRST;
+									   ? WheelSearchOrder.FIRST_TO_LAST
+									   : WheelSearchOrder.LAST_TO_FIRST;
```

Comment:
These two lines had tabs followed by spaces for a reason (tabs for indentation, spaces for alignment), you might want to check smart tabs in your formatting options. Please revert these back to spaces.

Reply:
Good catch! It would be helpful if you added the appropriate code style config to git; I use very different settings, so I created a project-specific profile and adjusted it as I noticed deviations; must have missed this.

Reply:
Is there an easy way to export IDEA settings into an `.editorconfig` or something?

Reply:
I actually thought that would be possible, but a cursory glance left me thinking .editorconfig is nowhere near as detailed as idea's config; it probably wouldn't allow for the indentation style you use in the first place (align to expression start; 0 continuation indent)

Reply:
Either way, this PR is not the time and place for it ^^

Reply:
Well, the indentation style isn't set in stone, it's mostly the best approximation I could achieve with IDEA's settings (they are far from covering everything, actually missing some really basic indentation settings that I want), so as long as most stuff is reasonably exported it's fine.